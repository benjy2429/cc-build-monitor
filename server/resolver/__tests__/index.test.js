import resolve from '../index';
import mockFetch from '../../mock-fetch';

describe('Resolve', () => {
  it('resolves and returns data', async () => {
    const result = await resolve(mockFetch);
    expect(JSON.parse(result)).toMatchSnapshot();
  });
});
