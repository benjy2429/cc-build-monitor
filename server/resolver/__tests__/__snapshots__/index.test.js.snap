// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resolve resolves and returns data 1`] = `
Object {
  "projects": Array [
    Object {
      "activity": "Sleeping",
      "buildNumber": "1",
      "buildStatus": "Success",
      "buildTime": "2000-01-01T12:00:00.000Z",
      "name": "project1",
    },
    Object {
      "activity": "Sleeping",
      "buildNumber": "13",
      "buildStatus": "Failure",
      "buildTime": "2000-01-01T12:00:00.000Z",
      "name": "project2",
    },
    Object {
      "activity": "Building",
      "buildNumber": "42",
      "buildStatus": "Unknown",
      "buildTime": "2000-01-01T12:00:00.000Z",
      "name": "project3",
    },
  ],
}
`;
